k8s/edge/Dockerfile

```diff
@@ -28,13 +28,10 @@ RUN set -ex \
   && grep "dumb-init_${DUMB_INIT_VERSION}_amd64$" sha256sums | sha256sum -c \
   && rm sha256sums \
   && mv dumb-init_${DUMB_INIT_VERSION}_amd64 /usr/bin/dumb-init \
-  && chmod +x /usr/bin/dumb-init \
-  && python3 -m pip install argparse jinja2 kubernetes urllib3 aionotify==0.2.0 \
-  && python3 -m pip install --upgrade pip==9.0.3 \
-  && python3 -m pip install "Pillow==5.0.0" \
-  && python3 -m pip install "numpy==1.14.3" \
-  && python3 -m pip install "pandas==0.23.0"
-# TODO: Remove Pillow, and pandas installation after toolchan system merge
+  && chmod +x /usr/bin/dumb-init
+
+COPY requirements.txt /root/requirements.txt
```

Comment:
@kirillmakhonin please, add comments that this is temporal approach to decrease build time and the source of requirements.txt - legion/requirements/base.txt that is copied here during Jenkins pipeline execution (Jenkinsfile) This workaround should be removed as soon as #226 is done